roles/restore/templates/management-pod.yaml.j2

```diff
@@ -14,7 +14,7 @@ metadata:
     app.kubernetes.io/managed-by: '{{ deployment_type }}-operator'
     owner: pulp-dev
 spec:
-  serviceAccountName: '{{ deployment_type }}-operator-sa'
+  serviceAccountName: '{{ deployment_name }}'
```

Comment:
```
 TASK [Create management pod from templated deployment config] ******************************** 
  fatal: [localhost]: FAILED! => {"changed": false, "error": 403, "msg": "Pod ci-pulprestore-backup-manager: Failed to create object: b'{\"kind\":\"Status\",\"apiVersion\":\"v1\",\"metadata\":{},\"status\":\"Failure\",\"message\":\"pods \\\\\"ci-pulprestore-backup-manager\\\\\" is forbidden: error looking up service account pulp-operator-system/example-pulp: serviceaccount \\\\\"example-pulp\\\\\" not found\",\"reason\":\"Forbidden\",\"details\":{\"name\":\"ci-pulprestore-backup-manager\",\"kind\":\"pods\"},\"code\":403}\\n'", "reason": "Forbidden", "status": 403}
```
https://github.com/pulp/pulp-operator/runs/8251041043?check_suite_focus=true#step:17:7344

maybe you need to remove the SA ownerreference

Reply:
ahh, I thought I had made that change.  Turns out I never pushed to the branch though.. Thanks!